### DISCUSSION TITLE
静态过滤规则语法

### DISCUSSION THREAD

**Discussion by SJinfinite** [uBlockOrigin/uAssets#23311] [Q&A] [answered]
静态过滤规则中，类似：
abc.com##.pageWrapper_AfHys > div:nth-of-type(4)
abc.com##.pageWrapper_Ujhgs > div:nth-of-type(4)
abc.com##.pageWrapper_Lhahs > div:nth-of-type(4)
请问应该如何用什么语法来涵盖表示上述的全部情况.

**Comment by gorhill** [comment, marked as answer]
abc.com##[class*="pageWrapper_"] > div:nth-of-type(4)

**Reply by SJinfinite** [reply]
十分感谢你的回答,问题已经成功解决了.
